masterlist.yaml

```
@@ -15004,7 +15004,7 @@ plugins:
         condition: 'active("ELE_SSE.esp") and not active("OCW_(ELE|ELEA)_FEPatch.esp") and not version("OCW_Obscure''s_CollegeofWinterhold.esp", "1.5.00", >=)'
       - <<: *patchIncluded
         subs: [ 'Enhanced Lights and FX' ]
-        condition: 'active("EnhancedLightsandFX.esp") and not active("OCW_ELFX_FEPatch.esp")'
+        condition: 'active("EnhancedLightsandFX.esp") and not active("OCW_ELFX_FEPatch.esp") and not active("UltimateCollege_ELFX_Patch.esp")'
```

It would probably be best to always hide these messages if `UltimateCollege.esp`is installed.
```suggestion
        condition: 'active("EnhancedLightsandFX.esp") and not active("OCW_ELFX_FEPatch.esp") and not active("UltimateCollege.esp")'
```

---

I'll change it to that